lib/ansible/collections/__init__.py

```diff
@@ -8,7 +8,7 @@
 
 from ansible.module_utils._text import to_bytes
 
-B_FLAG_FILES = frozenset([b'MANIFEST.json', b'galaxy.yml'])
+B_FLAG_FILES = frozenset([b'MANIFEST.json', b'galaxy.yml', b'galaxy.yaml'])
```

Comment:
iirc we had specifically stated that .yaml was an invalid extension for galaxy file.

Reply:
@bcoca the thing is that galaxy CLI treats it as valid. So we should probably be consistent here too.

Reply:
I wasn't aware of that. I guess we could deprecate `galaxy.yaml` for ansible-galaxy.

Reply:
@nitzmahone to confirm, since my info might be out of date